.jscsrc

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Does Mozilla have a specific company wide style for this sort of stuff? (mostly just curious)